lib/danger_plugin.rb

```diff
@@ -1,29 +1,37 @@
 module Danger
+  # Validates the code coverage of the files changed within a Pull Request.
+  # This method accepts the same arguments accepted by the xcov gem.
+  #
+  # @example Validating code coverage for EasyPeasy (easy-peasy.io)
+  #
+  #  xcov.report(
+  #    scheme: 'EasyPeasy',
+  #    workspace: 'Example/EasyPeasy.xcworkspace',
+  #    exclude_targets: 'Demo.app',
+  #    minimum_coverage_percentage: 90
+  #  )
+  #
+  #  # Checks the coverage for the EasyPeasy scheme within the specified
+  #  workspace, ignoring the target 'Demo.app' and setting a minimum
+  #  coverage percentage of 90%.
+  #  The result is sent to the pull request with a markdown format and
+  #  notifies failure if the minimum coverage threshold is not reached.
+  #
+  # @see: https://github.com/nakiostudio/danger-xcov
+  # @tags: xcode, coverage, xccoverage, tests, ios, xcov
   class DangerXcov < Plugin
```

Comment:
Awesome @orta thanks for your feedback. Just updated the PR removing the `:` and the `@see` linking to the repo 🍻